### DISCUSSION TITLE
VScode syntax error

### DISCUSSION THREAD

**Discussion by anas031** [AammarTufail/machinelearning_ka_chilla#10] [Q&A]
![Capture](https://user-images.githubusercontent.com/100215810/158534592-6b5b84b4-bcdd-4441-b9f4-7ef49a5186dc.PNG)
In DS101 lecture
@[6:03:00](https://www.youtube.com/watch?v=KbmMs1xj3vQ&t=21780s&fbclid=IwAR09c6VFePcrGwzHyTfegaDqgRolGQspdGLz2GDdF28dGJn41uvp_NkVyog)
QUESTION: I am trying to install pandas using pip install pandas in VSCode ipynb but it keeps giving an invalid syntax error. I tried to troubleshoot but couldn't. Can anyone help me with this?

**Comment by Faizali100** [comment]
Write it in terminal window showing in last, not in coding windows.

**Reply by anas031** [reply]
The issue was that I was installing two libraries i.e pandas and numpy in a single cell. I tried doing it in different cells and its working now, but thanks :) 
![Captuure](https://user-images.githubusercontent.com/100215810/158564663-fb263de6-93fb-4980-a7be-0854e966f91e.PNG)

**Reply by taqihaider7** [reply]
also update pip

**Reply by AammarTufail** [reply]
update pip as mentioned in the last line warning inside the single quotation marks '   '
